entries/jQuery.widget.xml

```diff
@@ -192,6 +192,21 @@
 			<xi:include href="../includes/widget-option-disabled.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 			<xi:include href="../includes/widget-option-hide.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 			<xi:include href="../includes/widget-option-show.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
+			<option name="classes" default="{}" example-value='{ "custom-header": "icon-warning" }'>
+				<desc>
+					<p>Extra classes to add to the widget. See the <a href="#method-_addClass"><code>_addClass()</code> method</a> for using this in custom widegts. Check out the documentation of individual widgets to see which classes they support.</p>
+					<p>The primary motivation of this option is to map structural classes to theme classes. In other words, any class prefixed with namespace and widget, like <code>"ui-accordion-"</code>, is considered a structural class. These are always added to the widget. In contrast to that, any class not specific to the widget is considered a theme class. These could be part of jQuery UI's CSS framework, but can also come from other CSS frameworks or be defined in custom stylesheets.</p>
+					<ul>
+						<li>structurual classes: the keys of this option, equivalent to the <code>keys</code> argument to <code>_addClass()</code>, for example <code>"ui-accordion-header"</code></li>
+						<li>theme classes: the values of this option, for example <code>"ui-corner-top"</code></li>
+					</ul>
```

Comment:
This seems a bit out of place in this format. Perhaps just add to the introductory paragraph instead. Something like "The structural classes are used as keys to the `classes` option and the thematic classes are the values."